src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
React doesn't need to be imported. It is imported by default.
fix it everywhere